src/status_im/signals/core.cljs

```diff
@@ -96,52 +96,52 @@
-      "backup.performed"           {:db (assoc-in db
-                                         [:profile/profile :last-backup]
-                                         (.-lastBackup event-js))}
-      "envelope.sent"              (transport.message/update-envelopes-status cofx
-                                                                              (:ids
-                                                                               (js->clj event-js
-                                                                                        :keywordize-keys
-                                                                                        true))
-                                                                              :sent)
-      "envelope.expired"           (transport.message/update-envelopes-status cofx
-                                                                              (:ids
-                                                                               (js->clj event-js
-                                                                                        :keywordize-keys
-                                                                                        true))
-                                                                              :not-sent)
-      "message.delivered"          (let [{:keys [chatID messageID]} (js->clj event-js
-                                                                             :keywordize-keys
-                                                                             true)]
-                                     (models.message/update-db-message-status cofx
-                                                                              chatID
-                                                                              messageID
-                                                                              :delivered))
-      "mailserver.changed"         (mailserver/handle-mailserver-changed cofx (.-id event-js))
-      "mailserver.available"       (mailserver/handle-mailserver-available cofx (.-id event-js))
-      "mailserver.not.working"     (mailserver/handle-mailserver-not-working cofx)
-      "discovery.summary"          (summary cofx (js->clj event-js :keywordize-keys true))
-      "mediaserver.started"        {:db (assoc db :mediaserver/port (.-port event-js))}
-      "wakuv2.peerstats"           (wakuv2-peer-stats cofx (js->clj event-js :keywordize-keys true))
-      "messages.new"               (transport.message/sanitize-messages-and-process-response cofx
-                                                                                             event-js
-                                                                                             true)
-      "wallet"                     (ethereum.subscriptions/new-wallet-event cofx
+      "node.login"              (profile.login/login-node-signal cofx (transforms/js->clj event-js))
```

Comment:
FYI: I had to reformat this file because somehow it got merged into `develop` incorrectly.